config/cesm/config_archive.xml

```diff
@@ -1,10 +1,10 @@
 <components version="2.0">
 
   <comp_archive_spec compname="cam" compclass="atm">
-    <rest_file_extension>\.[ri]\..*</rest_file_extension>
-    <rest_file_extension>\.rh\d\.*</rest_file_extension>
-    <rest_file_extension>\.rs\.*</rest_file_extension>
-    <hist_file_extension>\.h.*.nc$</hist_file_extension>
+    <rest_file_extension>[ri]</rest_file_extension>
+    <rest_file_extension>rh\d*</rest_file_extension>
+    <rest_file_extension>rs</rest_file_extension>
+    <hist_file_extension>[eh]</hist_file_extension>
```

Comment:
What is the 'e' for?

Reply:
Do you have an example where ESP files are showing up where CAM or CLM files belong?